traffic_ops/testing/api/v3/user_test.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func TestUsers(t *testing.T) {
-	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, DeliveryServices, Users}, func() {
+	WithObjs(t, []TCObj{Tenants, Users}, func() {
```

Comment:
This API test is failing for me.

```shell
[zrhoffman@computer api]$ time go test './v3' -test.v -cfg=$PWD/conf/traffic-ops-test.conf -fixtures=$PWD/v3/tc-fixtures.json -test.run 'TestMain|TestUsers'
INFO: traffic_ops_test.go:71: 2020-05-18T10:58:09.505714571-06:00: Using Config values:
                           TO Config File:       /home/zhoffm468/go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api/conf/traffic-ops-test.conf
                           TO Fixtures:          /home/zhoffm468/go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api/v3/tc-fixtures.json
                           TO URL:               https://localhost:6443
                           TO Session Timeout In Secs:  60
                           DB Server:            localhost
                           DB User:              traffic_ops
                           DB Name:              traffic_ops
                           DB Ssl:               false
=== RUN   TestUsers
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:58: Response:  {[{user was created. success}]}
    TestUsers: user_test.go:155: could not register user: 500 Internal Server Error[500] - Error requesting Traffic Ops https://localhost:6443/api/3.0/users/register {"alerts":[{"text":"Internal Server Error","level":"error"}]}
--- FAIL: TestUsers (2.03s)
FAIL
FAIL    github.com/apache/trafficcontrol/traffic_ops/testing/api/v3     2.320s
FAIL

real    0m3.824s
user    0m3.856s
sys     0m0.523s
```

Reply:
Ok that is the SMTP one...lemme turn on the external tests and look into that. Guessing maybe there is a data dependency I removed that is causing this to fail.

Reply:
Ok should be resolved now in https://github.com/apache/trafficcontrol/pull/4692/commits/46d204d80fb8644fb4345d7ddc504714ab67dd66